api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -228,4 +230,26 @@ public OrderGroup getParentOrderGroup() {
 	public void setParentOrderGroup(OrderGroup parentOrderGroup) {
 		this.parentOrderGroup = parentOrderGroup;
 	}
+
+	/**
+	 * Gets the order group reason which denotes the reason why the group was
+	 * ordered
+	 * 
+	 * @param Returns the order group reason
+	 * @since 2.4.0
+	 */
+	public Concept getOrderGroupReason() {
+		return orderGroupReason;
+	}
+
+	/**
+	 * Sets the order group reason which denotes the reason why the group was
+	 * ordered
+	 * 
+	 * @param order group reason, the order group reason to set
```

Comment:
The param annotation should have the exact param name which is being documented

Reply:
i have fixed it